superset/db_engine_specs/base.py

```diff
@@ -513,7 +510,7 @@ def create_table_from_excel(  # pylint: disable=too-many-arguments
         Create table from contents of a excel. Note: this method does not create
         metadata for the table.
         """
-        df = cls.excel_to_df(filepath_or_buffer=filename, **excel_to_df_kwargs,)
+        df = cls.excel_to_df(io=filename, **excel_to_df_kwargs,)
         engine = cls.get_engine(database)
```

Comment:
the first arg for read_excel is io, not filepath_or_buffer, which is for read_csv